rb/lib/selenium/webdriver/remote/http/common.rb

```diff
@@ -26,10 +26,18 @@ class Common
           CONTENT_TYPE    = 'application/json'
           DEFAULT_HEADERS = {
             'Accept' => CONTENT_TYPE,
-            'Content-Type' => "#{CONTENT_TYPE}; charset=UTF-8",
-            'User-Agent' => "selenium/#{WebDriver::VERSION} (ruby #{Platform.os})"
+            'Content-Type' => "#{CONTENT_TYPE}; charset=UTF-8"
           }.freeze
 
+          class << self
+            attr_accessor :extra_headers
+            attr_writer :user_agent
+
+            def user_agent
```

Comment:
Hmm, do we need all this? Watir is just subclassing the Default client and making a tweak here. https://github.com/watir/watir/blob/main/lib/watir/http_client.rb

What is Appium doing now?

Reply:
@KazuCocoa Would implementing a custom HTTP client in Appium be better than this?

Reply:
Oh, the watir way seems good. Appium overrides DEFAULT_HEADERS, b ut the watir way is sufficient for the usage. Then, this pr can drop the user_agent related change actually

Reply:
To be clear, I'm not saying the Watir approach is "correct," if it makes more sense to do something else, Watir will do that way. 😄  Presumably we are supposed to avoid using `Http::Default` in case someone wants to use `Http::Curb`? I don't really understand why we have that one. 😂

Reply:
Yea. Actually, the Watir way is better than the current Appium's one at least :P

Hmm, I like this pr approach, to provide a way to extend headers explicitly, than overriding the parent class and calling super

Reply:
Hmm, yes, I agree. With the approach in this PR we can just inject what the user sends regardless. Would be better to minimize subclassing. 

@twalpole does capybara do anything to the user agent?